client/src/leap/soledad/client/encdecpool.py

```diff
@@ -212,29 +174,7 @@ def enqueue_doc_for_encryption(self, doc):
         :param doc: The document to be encrypted.
```

Comment:
At some point we might need to rename this method, but let's keep it like that for the moment.